x-pack/test/security_solution_cypress/cypress/tasks/entity_analytics.ts

```diff
@@ -31,7 +31,8 @@ import { REFRESH_BUTTON } from '../screens/security_header';
 export const updateDashboardTimeRange = () => {
   // eslint-disable-next-line cypress/no-force
   cy.get(GET_DATE_PICKER_APPLY_BUTTON(GLOBAL_FILTERS_CONTAINER)).click({ force: true }); // Force to fix global timerange flakiness
-  cy.get(REFRESH_BUTTON).click();
+  // eslint-disable-next-line cypress/no-force
+  cy.get(REFRESH_BUTTON).click({ force: true }); // Force to fix even more global timerange flakiness
```

Comment:
very rarely (1 in over 1000 test runs) the query button is disabled I dont know why, but the button is not what is primarily under test so I think a force is reasonable here